tensorboard/uploader/uploader_main.py

```diff
@@ -409,6 +409,8 @@ def execute(self, server_info, channel):
             url = server_info_lib.experiment_url(server_info, experiment_id)
             print(url)
             data = [
+                ("Name", experiment.name),
+                ("Description", experiment.description),
```

Comment:
If you do make this change, please change the `name.ljust(10)` padding
below such that `Description` doesn’t abut into the next column.